src/components/Attachments/AttachmentView/index.js

```diff
@@ -153,6 +159,15 @@ function AttachmentView({
     if ((_.isString(source) && Str.isPDF(source)) || (file && Str.isPDF(file.name || translate('attachmentView.unknownFilename')))) {
         const encryptedSourceUrl = isAuthTokenRequired ? addEncryptedAuthTokenToURL(source) : source;
 
+        const onPDFLoadComplete = (path) => {
+            if (isUsedInCarousel && reportActionID && path) {
```

Comment:
why is this? `isUsedInCarousel`, logged `pdfPaths` and it is not storing the paths anymore due to this condition.